src/Directory.Build.props

```diff
@@ -0,0 +1,34 @@
+<Project>
+
+  <PropertyGroup>
+    <VersionPrefix>3.0.0</VersionPrefix>
```

Comment:
Lost support for old frameworks, but maybe that's not worth bumping the major version?